flask_user/signals.py

```diff
@@ -36,6 +36,9 @@
 # Sent when a user logged out
 user_logged_out = _signals.signal('user.user_logged_out')
 
+# Sent when a user attempted to login and password verification failed
+user_password_failed = _signals.signal('user.password_failed')
```

Comment:
Looking a little closer, his other variables follow this pattern:

```user_<action>```

Our doesn't. To follow this pattern we should go with something like:

```user_failed_password_verification```

This is unfortunately a long name. I'm going to stick with ```user_password_failed```, submit a PR and leave a comment to see what he thinks.